docs/guides-references/manifests.md

```diff
@@ -16,6 +16,7 @@ metadata:
   annotations:
     # global flags for all images in the manifest
     # image flags override global flags
+    # example, hauler.dev/registry: will only prepend if the server reference is missing
```

Comment:
```suggestion
    # example: platform set globally, but not observed if set per image
    # example: registry set globally, but not observed if set per image
```

Reply:
Please add changes to `docs/guides-references/hauler-content/images.md` as well.